app/src/main/java/com/kodeco/android/countryinfo/ui/components/CountryInfoRow.kt

```diff
@@ -1,13 +1,46 @@
 package com.kodeco.android.countryinfo.ui.components
 
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Card
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.countryinfo.model.Country
+import com.kodeco.android.countryinfo.model.CountryFlags
+import com.kodeco.android.countryinfo.model.CountryName
 
 // TODO fill out CountryInfoRow
 @Composable
-fun CountryInfoRow() { }
+fun CountryInfoRow(country: Country) {
+    val name = country.commonName
+    val capital = country.capital?.get(0) ?: ""
+
+    Card(
+        modifier = Modifier.fillMaxWidth().padding(all = 5.dp)
+    ) {
+        Column(
+            modifier = Modifier.padding(all = 5.dp)
+        ) {
+            Text(text = "Name: $name")
+            Text(text = "Capital: $capital")
+        }
+    }
+}
 
 // TODO fill out the preview.
-@Preview
+@Preview(showBackground = true)
 @Composable
-fun CountryInfoRowPreview() { }
+fun CountryInfoRowPreview() {
+    val country = Country(
+        name = CountryName("Trinidad & Tobago"),
```

Comment:
```suggestion
        name = CountryName("Trinidad & Tobago"),
```

 You could consider using stringResource() this is a compose related method that allows you to pull in string resource IDs that you set from your strings.xml. This is useful in order to support localized string values in case there's multiple languages your app supports.